src/common/logger/src/logger_win.cpp

```diff
@@ -5,12 +5,14 @@
 
 #include <memory>
 
+#define EVENT_ID 1000
```

Comment:
this ID will remain fixed no matter what events of the agents are being logged?

Reply:
Initially, the ID will be unique for all the agent events.